internal/uploader.go

```diff
@@ -215,7 +215,7 @@ func (uploader *RegularUploader) UploadJSON(ctx context.Context, path string, da
 
 	errorGroup, _ := errgroup.WithContext(ctx)
 	errorGroup.Go(func() error {
-		err := json2.MarshalWrite(writer, data)
+		err := json.NewEncoder(writer).Encode(data)
```

Comment:
Is there any reason to rollback https://github.com/wal-g/wal-g/pull/2056 ?
If it breaks compilation - upgrade to 1.25 and add `GOEXPERIMENT=jsonv2` to your env (this part is not obvious).

Reply:
yup, this is bogus change, I'll keep json2